package.json

```diff
@@ -39,6 +39,7 @@
     "http-errors": "^1.7.1",
     "isomorphic-fetch": "^2.2.1",
```

Comment:
`isomorphic-fetch` itself should be removed from explicit dependencies then.

Reply:
@birdofpreyru 
Thank you for your comment.
I assumed that I should remove `isomorphic-fetch` from dependencies.
However I got the below error when I ran poller.js.

'''
Error: Cannot find module 'isomorphic-fetch'
'''

In addition that, I got the bellow message when I installed `omni-fetch` after removing `isomorphic-fetch`.

'''
$ npm install omni-fetch
npm WARN omni-fetch@0.2.3 requires a peer of isomorphic-fetch@^2 but none is installed. You must install peer dependencies **yourself**.
'''

I guess we should install `isomorphic-fetch` with `omni-fetch` manually.

Reply:
> I guess we should install `isomorphic-fetch` with `omni-fetch` manually.

I'm stupid - what exactly does this mean? Is this a one-time thing that needs to be done in the repository?  A one-time thing for those who happen to have the repository checked out before this change? If so what are the steps? (Or would https://github.com/ConsenSys/armlet/blob/master/Makefile#L19 work?)

Surely, it's not something each npm install user needs to do, right?

Reply:
He just says that we should keep `isomorphic-fetch` as a separate entry in `package.json`, as `omni-fetch` does not install it automatically, although relies on it.

Reply:
Ah. Thanks! So this is all fine with me and if @birdofpreyru is okay with it, and this package.json is set properly (which I think it currently is), then let's merge!

Reply:
Yeah, I am fine. I have not really tested it, nor have previous experience with `omni-fetch`, but the promise is that it works the same as `isomorphic-fetch`, just adds proxies support, so should be all good.

Reply:
Ok. I will test later and merge. Thanks everyone!

Reply:
> He just says that we should keep `isomorphic-fetch` as a separate entry in `package.json`, as `omni-fetch` does not install it automatically, although relies on it.

This understanding is correct.
I checked the installation guide of `omni-fetch` just in case and found we need to install not only `omni-fetch` but also  `isomorphic-fetch`.

Thx!
https://github.com/marionebl/omni-fetch#installation